src/components/usa-button/content/usa-button~outline-inverse.json

```diff
@@ -1,5 +1,6 @@
 {
   "modifier": "usa-button--outline usa-button--inverse",
   "title": "Outline inverse button",
```

Comment:
The other stories have a "text" control instead of "title"

Reply:
This resolves the earlier mentioned button outline `is_demo` bug!